aptos-move/aptos-aggregator/src/transaction.rs

```diff
@@ -61,14 +62,14 @@ impl TransactionOutputExt {
     pub fn into_transaction_output(
```

Comment:
This function is also used in VM adapter (sequential) and `api/src/transactions.rs`. Do we want to clone materialized deltas there as well?

Reply:
let's see if we can refactor tomorrow - anyway we have to discuss Sasha's concern and probably apply this diff on top of yours semi-manually to be careful